contracts/modules/SimpleUpgrader.sol

```diff
@@ -53,6 +53,7 @@ contract SimpleUpgrader is BaseModule {
         uint256 i = 0;
         //add new modules
         for (; i < toEnable.length; i++) {
+            require(registry.isRegisteredModule(toEnable[i]), "SU: module is not registered");
```

Comment:
Is this PR needed because:
1) we want the code to strictly enforce the constraint that "a module must be registered prior to being authorised by a wallet"
or 2) we want to catch the scenario where we forget to register modules that are being added via a registered upgrader ?

If 2), then we could do the check above in the constructor of `SimpleUpgrader` to save some gas.